examples/ssh/mysql.js

```diff
@@ -37,45 +78,21 @@ function validate() {
  * @documentation This procedure is used for retrieving device * variables data
  */
 function get_status() {
-    D.device.sendSSHCommand(
-        {
-            command: command,
-            username: D.device.username(),
-            password: D.device.password(),
-            timeout: 20000
-        },
-        function (output, error) {
-            if (error) {
-                console.error(error);
-                D.failure();
-            } else {
-                var data = parse(output);
-                reportData(data);
-                if (!isRunning(data)) {
-                    tryStart();
-                }
-            }
+    D.device.sendSSHCommand(sshConfig,function (output, error) {
```

Comment:
dupplicate call for the same config, create a function for the ssh call that uses promises